src/js/components/NameValueList/stories/Layout.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
-import { Box, Grommet, NameValueList, NameValuePair, Text } from 'grommet';
+import {
+  Box,
+  Grommet,
+  Heading,
+  NameValueList,
+  NameValuePair,
+  Text,
+} from 'grommet';
 import { grommet } from 'grommet/themes';
 import { data, metricData } from './data';
 
 export const Layout = () => (
   <Grommet theme={grommet}>
     <Box pad="small" gap="large">
       <>
-        <Text weight="bold" size="2xl">
+        <Heading level={2} weight="bold" s>
```

Comment:
Since there isn't an `h1` on this page, I don't think it's semantically correct to have `h2`s -- wondering if we can just leave this story as is

Reply:
okay thats fine we can leave it as is